.circleci/config.yml

```diff
@@ -49,19 +49,24 @@ jobs:
            echo "import KarhooSDK" > Client/Keys.swift
            echo "import KarhooUISDK" >> Client/Keys.swift
            echo "struct Keys {" >> Client/Keys.swift
-           echo "static let identifier = \"\"" >> Client/Keys.swift
+           echo "static let braintreeGuestIdentifier = \"\"" >> Client/Keys.swift
```

Comment:
We dont need this echos anymore 
We can do something similar to this
https://github.com/karhoo/karhoo-android-ui-sdk/blob/develop/.circleci/config.yml#L36